tests/denops/runtime/functions/denops/notify_test.ts

```diff
@@ -38,23 +40,51 @@ testHost({
       // Load plugin and wait.
       await host.call("execute", [
         "let g:__test_denops_events = []",
-        `call denops#plugin#load('dummyLoaded', '${scriptValid}')`,
+        `call denops#plugin#load('dummy', '${scriptDispatcher}')`,
       ], "");
       await wait(async () =>
         (await host.call("eval", "g:__test_denops_events") as string[])
-          .includes("DenopsPluginPost:dummyLoaded")
+          .includes("DenopsPluginPost:dummy")
       );
 
-      outputs = [];
-      await host.call("denops#notify", "dummyLoaded", "test", ["foo"]);
+      await t.step("returns immediately", async () => {
+        await host.call("execute", [
+          "let g:__test_denops_events = []",
+        ], "");
 
-      await t.step("returns immediately", () => {
-        assertEquals(outputs, []);
+        await host.call("denops#notify", "dummy", "test", ["foo"]);
+
+        assertEquals(await host.call("eval", "g:__test_denops_events"), []);
       });
 
       await t.step("calls dispatcher method", async () => {
-        await delay(ASYNC_DELAY);
-        assertStringIncludes(outputs.join(""), 'This is test call: ["foo"]');
+        await delay(100 + ASYNC_DELAY);
```

Comment:
Please explain what is `100` and what is `ASYNC_DELAY` and why this `delay` is needed in comments.

Reply:
I don't remember the detail but can we use `wait` instead of `delay`?

```ts
await wait(() => maybe(host.call("eval", "g:__test_denops_events"), isArray)?.length > 0)
```